db/fixtures/miq_product_features.yml

```diff
@@ -438,16 +438,24 @@
       :description: Remove Cloud Providers
       :feature_type: admin
       :identifier: ems_cloud_delete
-    - :name: Edit
-      :description: Edit a Cloud Provider
+    - :name: edit
+      :description: Edit a Cloud Providers
```

Comment:
`Edit a Cloud Providers` is grammatically incorrect...same for the other ones.

Reply:
thanks @Fryguy, corrected